src/sentry/sentry_metrics/client/snuba.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Dict, Optional, Sequence, Union
+
+from django.core.cache import cache
+
+from sentry import quotas
+from sentry.sentry_metrics.client.base import GenericMetricsBackend
+from sentry.sentry_metrics.configuration import IndexerStorage, UseCaseKey, get_ingest_config
+from sentry.sentry_metrics.consumers.indexer.processing import MessageProcessor
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+from sentry.testutils.cases import BaseMetricsTestCase as metrics_test_base
```

Comment:
nit: please don't rename this class to snake_case on import

Reply:
done